### PR TITLE
Updating landing page deployed to Cloud Run to show next steps with Cloud Code

### LINKED ISSUE #274: [BUG] Update next steps on deployed page for Cloud Run samples to be relevant to Cloud Code
**Describe the bug**
The "next steps" once the sample is deployed is a "Build and deploy quickstart" that has nothing to do with Cloud Code. 

We should update this sample to have next steps that guide them to try more functionality in Cloud Code with this sample such as debugging (if local), viewing Cloud Run Explorer in the IDE, etc.

**To Reproduce**
1. Create a Cloud Run NodeJS sample application in Cloud Code (Command Palette > New Application > Cloud Run > NodeJS Hello World) 
2. Click on Cloud Code on the taskbar and select Cloud Run: Deploy
3. Select Deploy
4. Once the deploy succeeds, click the URL to view the website

**Expected behavior**
Next steps that relate back to this particular sample and using more functionality in Cloud Code

**Screenshots**
image

**Template in use:**
 - NodeJS
 - Cloud Run Hello World 

**Cloud Code in use:**
 - Cloud Code for VS Code
 - OS: OSX
 - Browser: Chrome
 - Version: 80

**Additional context**

### PR DESCRIPTION
**Pull request by simonz130**
fixes #274

### CODE REVIEW

File: dotnet/dotnet-cloud-run-hello-world/Pages/Index.cshtml [hunk 1]
```diff
@@ -36,10 +36,10 @@
     </p>
 
     <h3>What's next?</h3>
-    <p class="next">Follow the Quickstart tutorial to build a “Hello World” application in your favorite
-        language into
-        a container image, and then deploy it to Cloud Run.</p>
+    <p class="next">Manage Cloud APIs and libraries in Cloud Code for <a href="https://cloud.google.com/code/docs/intellij/client-libraries" target="_blank" rel="noopener">IntelliJ</a> and <a href="https://cloud.google.com/code/docs/vscode/client-libraries" target="_blank" rel="noopener">VSCode</a></p>
```
Comment (etanshaul) [inline dotnet/dotnet-cloud-run-hello-world/Pages/Index.cshtml:39]: is it just arbitrary that we are directing to Cloud APIs for IntelliJ, and View Logs for VS Code?

also, do we want to get rid of VS Code references from this as well?

Comment (simonz130) [reply dotnet/dotnet-cloud-run-hello-world/Pages/Index.cshtml:39]: We direct to Cloud APIs for both IntelliJ and VSCode.
As to Logs Viewer - I wanted to point to both extensions but then there was only logs viewer for VSCode.
We can hold VSCode references until they release Cloud Run to be consistent

Comment (etanshaul) [reply dotnet/dotnet-cloud-run-hello-world/Pages/Index.cshtml:39]: sgtm

Comment (etanshaul) [reply dotnet/dotnet-cloud-run-hello-world/Pages/Index.cshtml:39]: sgtm

Comment (simonz130) [reply dotnet/dotnet-cloud-run-hello-world/Pages/Index.cshtml]: Updated the code

Comment (simonz130) [reply dotnet/dotnet-cloud-run-hello-world/Pages/Index.cshtml]: etanshaul@, could you please LGTM so I can merge this?

File: java/java-cloud-run-hello-world/README.md [hunk 2]
```diff
@@ -23,14 +23,6 @@ This sample demonstrates how to use the Cloud Code extension in IntelliJ.
 * [Install Cloud Code for IntelliJ](https://cloud.google.com/code/docs/intellij/install)
 * [Creating a new Cloud Run app](https://cloud.google.com/code/docs/intellij/quickstart-cloud-run)
 
-### IntelliJ Running locally
```
Comment (etanshaul) [inline java/java-cloud-run-hello-world/README.md:26]: you need to remove step 2 from the table of contents as well I think

**Pull request merged**